pkg/pdapi/pd_control.go

```diff
@@ -116,6 +148,11 @@ func PdClientURL(namespace Namespace, clusterName string, scheme string) string
 	return fmt.Sprintf("%s://%s-pd.%s:2379", scheme, clusterName, string(namespace))
 }
 
+// ClusterRefPDClientUrl builds the url of cluster pd client
+func ClusterRefPDClientUrl(namespace Namespace, clusterName string, scheme string, clusterDomain string) string {
+	return fmt.Sprintf("%s://%s-pd.%s.svc.%s:2379", scheme, clusterName, string(namespace), clusterDomain)
```

Comment:
We should check namespace, clusterDomain and return different urls.

Reply:
fixed